core/src/main/spotbugs/exclude-filter.xml

```diff
@@ -32,4 +32,9 @@
       <Package name="org.apache.accumulo.core.trace.thrift" />
     </Or>
   </Match>
+  <Match>
+    <!-- Must ignore these everywhere, because of a javac byte code generation bug -->
+    <!-- https://github.com/spotbugs/spotbugs/issues/756 -->
```

Comment:
Would it make sense to create a 2.1 issue for this?

Reply:
Not really. There's nothing actionable we can do about it. It looks unlikely spotbugs will work around it any time in the near future, and upstream Java says it's intended, and not a bug. So, it will very likely never get fixed. Either way, there's nothing we can do about it for the foreseeable future other than to ignore it.